dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPI.java

```diff
@@ -37,52 +34,35 @@
      * @throws DotDataException     An error occurred when accessing the data source.
      * @throws IOException
      */
-    public PageView getPageMetadata(final HttpServletRequest request, final HttpServletResponse response,
-                                    final User user, final String uri, final PageMode mode)
-            throws DotSecurityException, DotDataException;
```

Comment:
let's deprecate and add comment to use new one instead - and don't remove the old one yet